react-frontend/src/components/ArrowButton.css

```diff
@@ -0,0 +1,22 @@
+.arrow-button {
```

Comment:
Thank you for the arrow button! I will ensure that this is added to my code.

Reply:
yay!